src/Microsoft.AspNetCore.Identity/IdentityOptions.cs

```diff
@@ -11,6 +13,18 @@ namespace Microsoft.AspNetCore.Builder
     /// </summary>
     public class IdentityOptions
     {
+        public IdentityOptions(IEnumerable<IConfigureOptions<IdentityOptions>> configureOptions = null)
```

Comment:
Yeah well the POR is https://github.com/aspnet/Options/issues/122.  Basically implementors can choose between this pattern and the nice pattern.  @lodejard feels strongly that this is the proper way to do things for framework code, so alas no sugar for us...